### ISSUE TITLE
Opfor and Blufor friendly tags sometimes don't appear

### ISSUE THREAD

**Issue by mooneymoon** [A3Wasteland/ArmA3_Wasteland.Altis#400]
I did a search and did not see an issue regarding this so hopefully it has not been already addressed.

Sometimes when a player loads into Opfor and Blufor their friendly tags never appear on nearby players' huds. The friendly players will mistake them for enemies and shoot them, only to be punished.

It seems to happen randomly so there are no steps to reproduce that I know of.

**Comment by AgentRev** [comment]
This has been happening since forever. I think it's a bug with the engine reporting an incorrect player position, outside of render distance. I don't know how to detect or prevent it.

**Comment by mooneymoon** [comment]
Since when it happens multiple players can't see the tag, would looking at the person's coordinates in the player menu / _getpos player_ prove you're right? I could probably see if I can get someone to do that when it happens next.

Once we determine that's the cause we can know for sure it's unfixable I guess. And maybe even add to BIS's laundry list of issues in the tracker.

**Comment by AgentRev** [comment]
Yes probably, but you would have to compare with the perceived position e.g. what GPS position the player is located in.
